tests/mmptest/src/BindingProjectTests.cs

```diff
@@ -155,6 +155,8 @@ public void ShouldBuildWithoutErrors_AndLinkCorrectFramework (BindingProjectType
 
 				Assert.False (logs.Item1.Contains ("CS1685"), "Binding should not contains CS1685 multiple definition warning:\n" + logs.Item1);
 
+				Assert.False (logs.Item1.Contains ("MSB9004"), "Binding should not contains MSB9004 warning:\n" + logs.Item1);
```

Comment:
I know we all use english. But if we have a bot configured to spanish and we land translations, we are going to have issues :/ 

Nothing really to be changed in this PR, but I've got the feeling this is going to hit us in the future.

Reply:
We have about a million of these in mmp/msbuild ios-msbuild tests.

If nothing else, we can use environmental variables to force language to test this (and hopefully translation as well).